packages/react-liveness/src/components/FaceLivenessDetector/shared/Toast.tsx

```diff
@@ -1,28 +1,36 @@
 import * as React from 'react';
-import { Flex, View, ViewProps } from '@aws-amplify/ui-react';
+import { Flex, View, ViewProps, useTheme } from '@aws-amplify/ui-react';
 
 import { LivenessClassNames } from '../types/classNames';
 
 interface ToastProps extends ViewProps {
   variation?: 'default' | 'primary' | 'error';
   size?: 'medium' | 'large';
   heading?: string;
+  isInitial?: boolean;
 }
 
 export const Toast: React.FC<ToastProps> = ({
   variation = 'default',
   size = 'medium',
   children,
+  isInitial = false,
   ...rest
 }) => {
+  const { tokens } = useTheme();
   return (
     <View
       className={`${LivenessClassNames.Toast} ${LivenessClassNames.Toast}--${variation} ${LivenessClassNames.Toast}--${size}`}
-      maxWidth={{ base: '100%', small: '70%' }}
+      {...(isInitial && { backgroundColor: tokens.colors.background.primary })}
       {...rest}
     >
       <Flex className={LivenessClassNames.ToastContainer}>
-        <Flex className={LivenessClassNames.ToastMessage}>{children}</Flex>
+        <Flex
+          className={LivenessClassNames.ToastMessage}
+          {...(isInitial ? { color: tokens.colors.font.primary } : {})}
```

Comment:
Think this can just be pass `color={isInitial ? tokens.colors.font.primary : undefined}`